iidm/iidm-api/src/main/java/com/powsybl/iidm/network/VariantManager.java

```diff
@@ -67,6 +81,21 @@
      */
     void cloneVariant(String sourceVariantId, String targetVariantId);
 
+    /**
+     * Create or overwrite a variant by cloning an existing one.
+     *
+     * @param sourceVariantId the source variant id
+     * @param targetVariantId the target variant id list (the ones that will be created
+     * @param mayOverwrite indicate if the target can be overwritten if it is already existing
+     * @throws com.powsybl.commons.PowsyblException
+     *      *                         if the source variant is not found or if a variant with
+     *      *                         an id of targetStateIds already exists and the mayOverwrite
+     *      *                         parameter is set to false
+     */
+    default void cloneVariant(String sourceVariantId, String targetVariantId, boolean mayOverwrite) {
+        cloneVariant(sourceVariantId, targetVariantId);
```

Comment:
I'm not sure this code is compliant with the javadoc: you should probably add a try/catch here two.